spec/features/dashboard_spec.rb

```diff
@@ -59,6 +59,26 @@
       expect(page).to have_content(lucky_user.user.email)
       expect(page).to_not have_content(unlucky_user.user.email)
     end
+
+    scenario 'feed displays 5 recent events' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.